Packs/FidelisEndpoint/Integrations/FidelisEndpoint/FidelisEndpoint.py

```diff
@@ -0,0 +1,1753 @@
+            'EventType': event.get('eventType'),
+            'ParentID': event.get('parentId'),
+            'PID': event.get('pid'),
+            'User': event.get('user'),
+            'ProcessStartTime': event.get('processStartTime'),
+            'LocalIP': event.get('localIP'),
+            'LocalPort': event.get('localPort'),
+            'RemoteIP': event.get('remoteIP'),
+            'RemotePort': event.get('remotePort')
+        })
+
+        context.append({
+            'EventTime': event.get('eventTime'),
+            'EndpointName': event.get('endpointName'),
+            'EventType': event.get('eventType'),
+            'ParentID': event.get('parentId'),
+            'TargetID': event.get('targetId'),
+            'PID': event.get('pid'),
+            'ParentName': event.get('parentName'),
+            'Name': event.get('name'),
+            'Path': event.get('path'),
+            'Hash': event.get('hash'),
+            'User': event.get('user'),
+            'LocalIP': event.get('localIP'),
+            'LocalPort': event.get('localPort'),
+            'RemoteIP': event.get('remoteIP'),
+            'RemotePort': event.get('remotePort'),
+            'DnsQuestion': event.get('dnsQuestion'),
+            'DnsAnswer': event.get('dnsAnswer'),
+            'ProcessStartTime': event.get('processStartTime'),
+            'EventIndex': event.get('eventIndex'),
+            'IndexingTime': event.get('indexingTime'),
+            'EntityType': event.get('entityType'),
+            'StartTime': event.get('startTime')
+        })
+
+    entry_context = {'FidelisEndpoint.Query(val.PID && val.PID === obj.PID)': context}
+    human_readable = tableToMarkdown('Fidelis Endpoint query events result', contents, headers=headers,
+                                     removeNull=True)
+    return human_readable, entry_context, response
+
+
+def parse_timestamp(time_):
+    if len(time_[time_.rfind('.')+1:time_.rfind('Z')]) > 3:
+        new_time = time_[:time_.rfind('.') + 4] + 'Z'
+    return new_time
+
+
+def fetch_incidents(client: Client, fetch_time: str, fetch_limit: str, last_run: Dict) -> Tuple[List, Dict]:
+    # handle first time fetch
+    last_run = last_run.get('time')
+    if not last_run:
+        last_run, _ = parse_date_range(fetch_time, to_timestamp=True)
+
+    current_fetch = last_run
+    incidents = []
+    last_fetch_date_string = timestamp_to_datestring(last_run, '%Y-%m-%dT%H:%M:%S.%fZ')
+    start_time = parse_timestamp(last_fetch_date_string)
+    response = client.list_alerts(limit=fetch_limit, start_date=start_time)
+    alerts = response.get('data', {}).get('entities', [])
+    for alert in alerts:
+        alert_id = str(alert.get('id'))
+        incident = {
+            'name': f"Fidlie Endpoint alert: {alert_id}",
+            'occurred': alert.get('createDate'),
+            'rawJSON': json.dumps(alert)
+        }
+        incident_date = int(dateutil.parser.parse(alert.get('createDate')).timestamp())*1000
```

Comment:
you're getting `createDate` twice, once for `occurred` and once in order to convert it to timestamp
instead initialize a variable with it